lib/datadog/appsec/context.rb

```diff
@@ -10,7 +11,7 @@ class Context
       ActiveContextError = Class.new(StandardError)
 
       # TODO: add delegators for active trace span
-      attr_reader :trace, :span, :events
+      attr_reader :trace, :span, :events, :request_milestones
```

Comment:
I think we should not expose it to outside directly, but rather expose API to operate with it. Same as we don't expose WAF runner, but expose API to use it. It will allow to extend it if needed without changing the caller side.

Or we should give it a better name to emphasis on what it is actually is.